### PR TITLE
adding mock mug comm info

### PR DESCRIPTION
**Pull request by eilinora**
mock convo and comment list

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -87,4 +87,49 @@ export const MOCK_EVENT = {
...
+			"delete",
+			"comment",
+			"follow",
+			"flag_spam"
+		]
+	}
+};
+
 
+export const MOCK_GROUP_COMMENTS = {
```
Comment (mmcgahan) [inline src/api.js]: something's not right here - this is identical to `MOCK_GROUP_COMMUNICATION`.

If possible, could you add a link to API docs if there are any?

**Review by mmcgahan — DISMISSED** [review]

File: src/api.js (hunk 1 above)
Comment (eilinora) [reply src/api.js]: whoops!

**Comment by sadafie** [comment]
The actual changes in this LGTM, but now that I'm seeing this, I want to bring up an earlier thought.... api.js is going to be huge if we just keep appending to it.

How about (outside of this PR), we refactor `api.js` => `src/api/<feature/endpoint>.js`, etc., or some other structure?

File: src/app.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ import { Observable } from 'rxjs';
 import {
 	MOCK_GROUP,
 	MOCK_EVENT,
+	MOCK_GROUP_COMMUNICATION,
```
Comment (sadafie) [inline src/app.js:5]: do these need to be here?

**Comment by mmcgahan** [comment]
@eilinora do you know whether there are any API docs for this yet? If you could add a link next to the mock object that would be great - I suspect we'll probably want to require something like that in the future for this repo, but if there's not a good reference right now we can merge

**Comment by eilinora** [comment]
@mikem-mup unfortunately not yet for this call. As it was a quick dev turn around. I can update with the link once available

File: src/app.js (hunk 2 above)
Comment (eilinora) [reply src/app.js:5]: yeah, i was missing the additions to app state before, added below

**Pull request merged**